src/views/trash-panel/trash-panel.tsx

```diff
@@ -143,7 +143,7 @@ export const TrashPanel = withStyles(styles)(
 
             handleContextMenu = (event: React.MouseEvent<HTMLElement>, resourceUuid: string) => {
                 const kind = resourceKindToContextMenuKind(resourceUuid);
-                const resource = getResource(resourceUuid)(this.props.resources) as TrashResource;
+                const resource = getResource(resourceUuid)(this.props.resources) as TrashableResource;
```

Comment:
TrashableResource should be used as generic type parameter of getResource. `getResource<TrashableResource>(...)`